syntax = "proto3";

message StudentList {
    int32 student_id = 1;
    string student_name = 2;
    int32 age = 3;
    string address = 4;
}

message Response {
    string message = 1;
}

message StudentListMessagesGroup {
    repeated StudentList studentList = 1;
}
